docs/niscope/functions.rst

```diff
@@ -1241,7 +1241,7 @@ niscope.Session methods
         wfm (list of float): 
 
 
-            Returns an array whose length is the **numSamples** times number of waveforms. Call :py:meth:`niscope.Session._actual_num_wfms` to determine the number of waveforms.
+            Returns an array whose length is the **numSamples** times number of waveforms. Call niScope\_ActualNumWfms to determine the number of waveforms.
```

Comment:
This appears to be a regression. Probably has to do with the thing being private so it might be an improvement over a broken link. However, notice the backslash.

Reply:
I missed changing the hand maintained documentation to remove '\'.  Fixed

Reply:
> I missed changing the hand maintained documentation to remove '\'. Fixed

FYI, the backslash in your comment was not displayed because it is rendered as markdown. I edited this one comment by escaping , or you could make it look like code: `'\'`. It threw me off for a second there.